### DISCUSSION TITLE
How do a change the map to a workshop map?

### DISCUSSION THREAD

**Discussion by Rydez** [kus/cs2-modded-server#44] [Q&A] [answered]
This is great work. Thank you!

I'm wondering how to manually force a map change. For example, `!rcon changelevel de_nuke` works., But when in gungame, `!rcon changelevel fun_bounce` doesn't work.

I've also tried a few different things that didn't work. It appears that `!rtv` is disabled with a note that it doesn't work well with workshop maps.

**Comment by kus** [comment, marked as answer]
Hey it’s undocumented as I haven’t finished the map chooser plugin, but as an admin you can type `!map` and it will bring up a menu of all the maps for the current mod.
